protocol-designer/src/load-file/migration/8_5_0.ts

```diff
@@ -87,7 +87,9 @@ const getClippedFlowRateForMoveLiquid = (args: {
   const tipLiquidSpecs = liquidClass?.byPipette
     .find(byPipette => byPipette.pipetteModel === pipetteName)
     ?.byTipType.find(byTipType => byTipType.tiprack === tiprack)
+  // TODO: this won't find the definition for a custom tiprack
   const tiprackDef = getAllDefinitions()[tiprack]
+  console.assert(tiprackDef, `could not find labware definition for ${tiprack}`)
```

Comment:
just to be a bit more explicit 
```suggestion
  console.assert(tiprackDef != null, `could not find labware definition for ${tiprack}`)
```